andela/ah-code-blooded
# [Feature #160577623] Serves the article read time functionality

## What does this PR do?
Serves the article read time functionality `api/articles` endpoints 

## Pivotal Tracker Story
Users should be able to see time it takes to read an article.

## Tasks accomplished 

| Endpoint        | Functionality        | Branch  |Pivotal Tracker
| ------------- |-------------| ----------| -------------|
| api/articles/| View an article |      `ft-read-time-160577623`      |  Story is [here](https://www.pivotaltracker.com/n/projects/2198535/stories/160577623) |

## Manual Testing
`Run server` --> **python manage.py runserver**
`Postman` 
1. Create Article
`POST` on `http://127.0.0.1:8000/api/articles/` with:
``` 
{
   "article":
                {
		"title": "Article Readtime PR",
		"description": "The article readtime shows how long it takes to read",
		"body": "The article readtime is calcilated using TRT = NoW/275 + [(10/60)*NoI)"
		}
}
```
2. Get the Articles
When you get the articles using:
`GET` on `http://127.0.0.1:8000/api/articles/`
You expect to see the readtime of the article:
<img width="598" alt="screen shot 2018-10-12 at 14 50 02" src="https://user-images.githubusercontent.com/31407254/46867620-466a7700-ce2e-11e8-852b-e7a4d6623fc3.png">
